crates/evm/fuzz/src/strategies/mutators.rs

```diff
@@ -0,0 +1,482 @@
+use alloy_primitives::{Address, I256, Sign, U256};
+use proptest::{prelude::*, test_runner::TestRunner};
+use rand::seq::IndexedRandom;
+
+// Interesting 8-bit values to inject.
+static INTERESTING_8: [i8; 9] = [-128, -1, 0, 1, 16, 32, 64, 100, 127];
+
+/// Interesting 16-bit values to inject.
+static INTERESTING_16: [i16; 19] = [
+    -128, -1, 0, 1, 16, 32, 64, 100, 127, -32768, -129, 128, 255, 256, 512, 1000, 1024, 4096, 32767,
+];
+
+/// Interesting 32-bit values to inject.
+static INTERESTING_32: [i32; 27] = [
+    -128,
+    -1,
+    0,
+    1,
+    16,
+    32,
+    64,
+    100,
+    127,
+    -32768,
+    -129,
+    128,
+    255,
+    256,
+    512,
+    1000,
+    1024,
+    4096,
+    32767,
+    -2147483648,
+    -100663046,
+    -32769,
+    32768,
+    65535,
+    65536,
+    100663045,
+    2147483647,
+];
+
+/// Mutator that randomly increments or decrements an uint or int.
+pub(crate) trait IncrementDecrementMutator: Sized + Copy {
+    fn increment_decrement(self, size: usize, rng: &mut TestRunner) -> Option<Self>;
+}
+
+macro_rules! impl_increment_decrement {
+    ($($t:ty => $validate_fn:path),*) => {
```

Comment:
$validate_fn is unused

Reply:
oops, removed in https://github.com/foundry-rs/foundry/pull/11061/commits/864c9a6797ee1bde547ba8ec42f84b59353cf979 and made validation logic nicer / early return on same value